src/templates/staff/assignment.html

```diff
@@ -101,6 +101,7 @@
                             + '</td><td class="text-right">' + fmtTimestamp(result[i].end)
                             + '</td><td class="text-right">' + result[i].max_runs
                             + '</td><td class="text-right">' + result[i].remaining_runs
+                            + '</td><td class="text-right"><button id="btn-' + result[i]._id + '" class="btn btn-danger" onclick="deleteExtension(\'' + result[i]._id + '\')">Delete</button>'
```

Comment:
This buttons is kinda big. Let's use an icon instead. Something like `<i class="text-danger fas fa-trash-alt" style="cursor: pointer"></i>`.

Reply:
Would you rather the button look like this:
![image](https://user-images.githubusercontent.com/20424336/99318397-399a8780-282d-11eb-8878-8c8313c31ad7.png)
...or like this:
![image](https://user-images.githubusercontent.com/20424336/99318446-58008300-282d-11eb-8fb1-fe86ec213d06.png)

Reply:
I'm fine with both, up to you!